src/windowmanagement/WindowLayout/WindowLayoutProvider.tsx

```diff
@@ -67,18 +80,22 @@ export function WindowLayoutProvider({ children }: { children: React.ReactNode }
           else {
             tab.group = 'regularWindow';
             const panel: PanelData = {
-              tabs: [tab]
+              tabs: [tab],
+              size: 300
```

Comment:
Make a named variable for this? `SidePanelDefaultWidth` or something.  Could be global in this file?